crates/cxx-qt-gen/src/generator/cpp/invokable.rs

```diff
@@ -140,6 +140,25 @@ pub fn generate_cpp_invokables(
                 },
             },
         });
+
+        // Note that we are generating a header to match the extern "Rust" method
+        // in Rust for our invokable.
+        //
+        // CXX generates the source and we just need the matching header.
+        //
+        // TODO: this could be private ?
```

Comment:
This could be sorted out as part of #20 and in the changes with #566 might help.